test/server/models/Update.test.js

```diff
@@ -209,6 +216,13 @@ describe('server/models/Update', () => {
         expect(count).to.eq(0);
       });
 
+      it('returns 0 when the audience is NO_ONE', async () => {
+        const emptyCollective = await fakeCollective();
+        const update = await fakeUpdate({ CollectiveId: emptyCollective.id, notificationAudience: 'NO_ONE' });
```

Comment:
`emptyCollective` has no member, so this test doesn't serve its purpose. Let's use a collective with some data:

```suggestion
        const update = await fakeUpdate({ CollectiveId: collective.id, notificationAudience: 'NO_ONE' });
```